src/style.css

```diff
@@ -1 +1,74 @@
 /* Styles go here */
+
+body {
+  margin: 0;
+  background-color: #e5e5e5;
```

Comment:
Delete it. We don't need to get background-color.

Reply:
margin = 0 - its important

Reply:
I told about background-color )